website/src/main/resources/microsite/data/rules.yml

```diff
@@ -13,4 +13,4 @@ rules:
 - Disable
 - DisableSyntax
 - DisableUnless
-- LeakingSealed
```

Comment:
FYI we'll probably get rid of rules.yml now thanks to tut:passthrough, I will take a look in future PR